NDTensors/ext/NDTensorCUDA/adapt.jl

```diff
@@ -0,0 +1,25 @@
+#
+# Used to adapt `EmptyStorage` types
+#
+
+to_vector_type(arraytype::Type{CuArray}) = CuVector
```

Comment:
Can we remove `to_vector_type` and use `set_ndims(arraytype, 1)` instead?

Reply:
Yes!